spec/models/time_entry_spec.rb

```diff
@@ -351,5 +351,76 @@
         it { expect(TimeEntry.visible(time_entry2.user, project).all).to match_array([time_entry2]) }
       end
     end
+
+    context 'calculate dates' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.